docs/imaging/sync.rst

```diff
@@ -4,31 +4,117 @@
 
-.. figure:: /_static/imaging/wavesurfer_sync_example.png
+.. figure:: /_static/imaging/sync/wavesurfer_sync_example.png
    :alt: Synchronisation of parallel data streams
 
 Events are extracted according to the polarity of the digital signal - i.e. rising or falling edge - and shown on the image below as red bars. A *last_index_master* is inferred (since not actually recorded) and the other sync streams are cut accordingly. To concatenate multiple (sub-)sessions, sync streams are zeroed on master (*first_index_master*)  and every subsequent sync stream is concatenated to the previous one adding 1 to *last_index_master*.
 
-.. figure:: /_static/imaging/wavesurfer_sync-01.jpg
+.. figure:: /_static/imaging/sync/wavesurfer_sync-01.jpg
    :alt: Synchronisation of parallel data streams
 
+Wavesurfer stores its data in an ``hdf5`` container. A reader for these containers is provided under `/helpers/mini2p_wavesurfer.py <https://github.com/kavli-ntnu/dj-moser-imaging/blob/master/helpers/mini2p_wavesurfer.py>`_. 
+
 Scanimage sync
 ---------------------
 
-Data can be ``pre-synced`` with scanimage. That means that either scanimage is triggering a - for example - tracking camera for every frame it acquires or timestamp information is recorded at imaging clock rate on one of 4 ``AUX`` inputs. The timing info for every frame and also an array of time stamps at imaging clock precision (`AUX 0 to 3`) get saved in the header of every page of the scanimage tif file. 
\ No newline at end of file
+The OpenMiniscope system implements one particular special case of the more general synchronisation scheme outlined above. Data can be *pre-synchronised* with the `ScanImage <http://scanimage.vidriotechnologies.com/display/SIH/ScanImage+Home>`_ software. In this case, instead of each instrument operating at its own sampling frequency determined by its own independent clock, each frame of acquisition is triggered by a signal sent from a single "master" source, which is the device controlling the two-photon imaging acquisition. Each time that ScanImage triggers a frame of two-photon imaging data, it can send a trigger signal to other instruments, causing them to acquire a frame of data simultaneously, or at least with a known, constant, offset. 
+
+In this case, the timestamp data for two-photon imaging frames is identical to the timestamp data for tracking frames.
+
+Note that although this method is superficially much simpler than the more general case, it has its own complexities, especially where multi-plane two-photon imaging is in use. In general, the design of the OpenMiniscope system is to send one trigger to external instruments on each *volume* acquisition, rather than on each *plane* acquisition, and consequently, there may be an integer multiple between the number of plane acquisitions and the nnumber of tracking frames. 
+
```

Comment:
no, the tracking is triggered on each "plane", for example, if the volume rate is 7 Hz for 2 planes, the tracking camera runs at 14 Hz.